### PR TITLE
new package: ubase

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/ubase/depends [hunk 1]
```diff
@@ -0,0 +1 @@
+make
```
Comment (dylanaraps) [inline community/ubase/depends]: This file can be removed.

File: community/ubase/version [hunk 2]
```diff
@@ -0,0 +1 @@
+1 1
```
Comment (dylanaraps) [inline community/ubase/version]: Alpine versions `ubase` using dates. Current is `20190312`. Though this is rather uncertain as Alpine is the only distributions shipping a non `-git` package for `ubase` on Repology.

https://repology.org/project/ubase/versions

Comment (a-schaefers) [reply community/ubase/version]: i suggest we make the version to be the date of the hash we checkout from mforney's repos for both sbase and ubase @dylanaraps

Comment (dylanaraps) [reply community/ubase/version]: I'm happy either way. :+1:

Comment (a-schaefers) [reply community/ubase/version]: thanks

**Comment by dylanaraps** [comment]
A post install script mentioning the need to add `/opt/bla/bla` to your `PATH` would be nice too.

Example:

```
#!/bin/sh

cat <<EOF

NOTE: This package is installed to /opt/bla/bla.
NOTE: Modify your '$PATH' to include this directory.

-> export PATH="$PATH:/opt/bla/bla"

EOF
```

**Comment by a-schaefers** [comment]
1

> A post install script mentioning the need to add `/opt/bla/bla` to your `PATH` would be nice too.
> 
> Example:
> 
> ```
> #!/bin/sh
> 
> cat <<EOF
> 
> NOTE: This package is installed to /opt/bla/bla.
> NOTE: Modify your '$PATH' to include this directory.
> 
> -> export PATH="$PATH:/opt/bla/bla"
> 
> EOF
> ```
> 
> Edit: Same goes for `sbase` or other `/opt` packages.

We have removed /opt entirely already. There is no /opt

**Comment by a-schaefers** [comment]
ready for merge

**Pull request merged**